app.js

```diff
@@ -0,0 +1,22 @@
+const express = require("express");
+
+const app = express();
+const { getTopic } = require("./controllers/topic.controllers");
+
+app.use(express.json());
+
+app.get("/api/topics", getTopic);
+
+app.use((err, req, res, next) => {
```

Comment:
This function is currently not being used in any way. Try not to add code that does not do anything with the current functionality. How could a custom error handler fire for this end point when you have no catch block in your controller.